README.md

```diff
@@ -436,6 +490,25 @@ There are a few parameters that can help you customize the noise amount for diff
 
 Note that the target field should be of either a numeric type (integer, decimal, unsignedInt, positiveInt) or a quantity type (Quantity, SimpleQuantity, Money, etc.). 
 
+## Generalize method
+As one of the anonymization method, generalization means mapping values to the higher level of generalization. It is the process of abstracting distinguishing value into a more general, less distinguishing value. Generalization attempts to preserve data utility while also reducing the identifiability of the data. 
+Generalization uses FHIRPath predicate expression to define a set of cases that specify the condition and target value like [sample rules](#Sample-rules-using-FHIRPath). Follows are some examples of cases.
+
+|Data Type|Cases|Explanation|Input data-> Output data|
+|-----|-----|-----|-----|
+|numeric|_"$this>=0 and $this<20": "20"_|Data fall in the range [0,20) will be replaced with 20. |18 -> 20|
+|numeric|_"true": "($this div 10)*10"_|Approximate data to multiples of 10. |18 -> 10|
+|string| _"$this in ('es-AR' \| 'es-ES' \| 'es-UY')": "'es'"_|Data fall in the value set will be mapped to "es".|'es-UY' -> 'es'|
+|string| _"$this.startsWith(\'123\')": "$this.subString(0,2)+\'*\*\*\*\' "_ |Mask sensitive string code.|'1230005' -> '123****'|
+|date, dateTime, time|_"$this >= @2010-1-1": "@2010"_|Data fall in a date/time/dateTime range will be mapped to one date/time/dateTime value.| 2016-03-10 -> 2010|
+|date, dateTime, time|_"$this.replaceMatches('(?&lt;year&gt;\\\d{2,4})-(?&lt;month&gt;\\\d{1,2})-(?&lt;day&gt;\\\d{1,2})\\\b', '${year}-${month}'"_|Omit "day" to generalize specific date.|2016-01-01 -> 2016-01|
+
+For each generalization rule, there are several additional settings to specify in configuration files:
+- [required] **cases** An object defines key-value pairs to define case condition and replacement value using FHIRPath predicate expression. _key_ represents case condition and _value_ represents target value.

```

Comment:
An object defines key-value pairs to define -> An object defining key-value pairs to specify